common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -465,7 +465,10 @@ class @Problem
       saveMessage = response.msg
       if response.success
         @el.trigger('contentChanged', [@id, response.html])
-        @render(response.html, @focus_on_save_notification)
+        @el.find('.notification-save .notification-message').html(saveMessage)
+        @clear_all_notifications()
+        @saveNotification.show()
+        @focus_on_save_notification()
```

Comment:
I will look into this tomorrow morning.  If I merge them I will just update the name of the method to reflect that it is showing and focusing.  I personally like the separation right now because it matches the other uses of focus and show in the file and separates the two actions.